post-types/templates/page-profiles-list-page.php

```diff
@@ -9,6 +9,20 @@
       ?>
     </div>
   </div>
+  <?php
+    if (odm_language_manager()->get_current_language() == 'km') {
```

Comment:
@Huyeng this only applies for ODC site. What happens if ODMm want to use this functionality. This line should rather be:

```if (odm_language_manager()->get_current_language() != 'en') {```

Can you please change it in ALL places?